tests/steps/other_v2_steps.py

```diff
@@ -1414,3 +1418,33 @@ def transaction_proof(context, round, txid, hashtype):
 @when("we make a Lookup Block Hash call against round {round}")
 def get_block_hash(context, round):
     context.response = context.acl.get_block_hash(round)
+
+
+@when("I simulate the transaction")
+def simulate_transaction(context):
+    context.simulate_response = context.app_acl.simulate_transactions(
+        [context.stx], response_format="json"
+    )
+
+
+@then("the simulation should succeed")
+def simulate_transaction_succeed(context):
+    if hasattr(context, "simulate_response"):
+        assert context.simulate_response["would-succeed"] is True
```

Comment:
should succeed or would succeed

Reply:
I'm not sure I follow?

It's `would-succeed` in the api 
https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/algod.oas2.json#L4044

Reply:
sorry I also changed the step names upstream: https://github.com/algorand/algorand-sdk-testing/pull/265/files#diff-15dc2e739dbef5460705db37e5398ba985224f5ba449daa0a32dfdaaf1dfad49R30

current format: 
```    
    the simulation should succeed without any failure message
    And the simulation should report a failure at path "" with message ""
    ```